packages/plugins/GoPlusSecurity/src/SNSAdaptor/rules.ts

```diff
@@ -280,4 +280,38 @@ export const SecurityMessages: SecurityMessage[] = [
         messageKey: 'risk_is_whitelisted_body',
         shouldHide: (info: TokenSecurity) => info.is_whitelisted === undefined,
     },
+    // true token
+    {
+        type: 'info-security',
+        level: SecurityMessageLevel.Safe,
+        condition: (info: TokenSecurity) => info.is_true_token === '1',
+        titleKey: 'risk_is_true_token_title',
+        messageKey: 'risk_is_true_token_body',
+        shouldHide: (info: TokenSecurity) => info.is_true_token === undefined,
+    },
```

Comment:
```suggestion
    {
        type: 'info-security',
        level: SecurityMessageLevel.Safe,
        condition: matchesProperty('is_true_token', '1'),
        titleKey: 'risk_is_true_token_title',
        messageKey: 'risk_is_true_token_body',
        shouldHide: matchesProperty('is_true_token', undefined),
    },
```

see https://lodash.com/docs/4.17.15#matchesProperty